gradle.properties

```
@@ -1,5 +1,5 @@
 group=org.partiql
-version=1.0.0-rc.2
+version=1.0.0-rc.2-SNAPSHOT
```

Could also set the version to RC3-SNAPSHOT

```suggestion
version=1.0.0-rc.3-SNAPSHOT
```